tests/10apidoc/33room-members.pl

```diff
@@ -261,10 +261,15 @@
       my ( $do_request_json_for, $await_event_for ) = @_;
 
       provide make_test_room => sub {
-         my ( $creator, @other_members ) = @_;
+         # First member is the room creator
+         my ( $members, %options ) = @_;
+
+         $options{visibility} = "public" unless exists $options{visibility};
 
          my $room_id;
          my $room_alias_shortname = "test-$next_alias"; $next_alias++;
+         my $creator = @$members[0];
+         my @other_members = @$members[1 .. $#$members];
```

Comment:
Can be done neater using list unpack:

    my ( $creator, @other_members ) = @$members;